tests/foreman/virtwho/conftest.py

```diff
@@ -52,3 +52,51 @@ def test_foo(session):
 
     """
     return Session(test_name, module_user.login, module_user.password)
+
+
+@pytest.fixture(scope='module')
+def module_user_sca(request, module_target_sat, module_org, module_location):
+    """Creates admin user with default org set to module org and shares that
+    user for all tests in the same test module. User's login contains test
+    module name as a prefix.
+
+    :rtype: :class:`nailgun.entities.Organization`
+    """
+    # take only "module" from "tests.foreman.virtwho.test_module"
+    test_module_name = request.module.__name__.split('.')[-1].split('_', 1)[-1]
```

Comment:
do you know of cases where this wouldn't work?
```suggestion
    test_module_name = request.module.__name__.split('_', 1)[-1]
```